odbc/2.windows.md

```diff
@@ -0,0 +1,34 @@
+# Windows ODBC Setup
+
+This page will help you set up your ODBC environment for accessing Db2 directly from a Windows machine. If you want to connect to your IBM i from a [Linux](./2.linux.md) or an [IBM i](./2.ibmi.md), please consult those pages.
+
+## Driver Manager Installation
+
+Windows comes preinstalled with an ODBC driver manager. To access it, search for `Administrative Tools` on your system (either through the search bar, or `Control Panel > System and Security > AdministrativeTools`), and then from there select ODBC Data Sources (either 32-bit or 64-bit).
```

Comment:
One thing to note: On Windows, *System DSNs* must be defined per-architecture, while *User DSNs* are architecture agnostic.

So if you create a 64-bit System DSN, it will not be seen by a 32-bit application and vice-versa.